common/install_health_checks.sh

```diff
@@ -1,8 +1,11 @@
 #!/bin/bash
 set -ex
 
-# Set the azhc version
-azhc_version=$(jq -r '.azhc."'"$DISTRIBUTION"'".version' <<< $COMPONENT_VERSIONS)
+# grab latest release version of AZHC
+repo=Azure/azurehpc-health-checks
+release_version=$(curl -s "https://api.github.com/repos/$repo/releases/latest" | jq -r '.tag_name')
+
+AZHC_VERSION=$release_version
```

Comment:
in the spack version of the script, azhc_version is lowercase. please update here to match.

Reply:
also, you could set it directly (no need for release version) and it includes the "v", so remove that from the branch name below

Reply:
IMHO, it will still be a good idea to put the specific version in requirements.json, and we have a central place to control all the different versions.

Reply:
Rafael was the one to update this to always pull latest. Maybe we should discuss what the desired behavior is.

Reply:
> IMHO, it will still be a good idea to put the specific version in requirements.json, and we have a central place to control all the different versions.

Rafael was the one to update this to always pull latest. Maybe we should discuss what the desired behavior is.

Reply:
I've addressed the issue of case sensitivity and am pushing version directly to azhc_version.

I have not addressed the requirements.json question.  Please provide guidance on how I should proceed.

Reply:
Reverted changes to azhc grab latest.    Updated Requirements with latest azhc=0.2.9